client/layout/masterbar/checkout.tsx

```diff
@@ -108,7 +125,21 @@ const CheckoutMasterbar = ( {
 				<span className="masterbar__secure-checkout-text">{ translate( 'Secure checkout' ) }</span>
 			</div>
 			{ title && <Item className="masterbar__item-title">{ title }</Item> }
-			{ loadHelpCenterIcon && <DefaultMasterbarContact /> }
+
+			<div className="masterbar__item-help-icons">
+				{ loadHelpCenterIcon && <DefaultMasterbarContact /> }
+
+				<ChatButton
+					chatIntent="PRESALES"
+					initialMessage={ initialMessage }
+					siteUrl={ siteSlug }
+					className={ classnames( 'masterbar__item' ) }
+					onClick={ handleClick }
+				>
+					{ icon && <MaterialIcon icon={ icon } /> }
```

Comment:
I think that `icon` is always defined here so we can remove this condition.
```suggestion
					<MaterialIcon icon={ icon } />
```